src/schemas/documentSchema.ts

```diff
@@ -4,8 +4,8 @@ export const documentSchema = yup
   .object({
     family_import_id: yup.string().required(),
     variant_name: yup.string().optional(),
-    role: yup.string().required(),
-    type: yup.string().required(),
+    role: yup.string().optional(),
```

Comment:
These fields are not optional, unless for MCF data.

Reply:
I have moved the validation for the data into the form itself and out of Yup `isRequired={!!taxonomy?._document?.role}`, keeping it strictly required here is not necessarily correct as we are applying to all schema (MCFs, UNFCCC etc) and as you note its not required for MCF data @katybaulch

Reply:
My concern with this is the inconsistency with our existing implementation -- the yup schema is meant to be the source of truth (I believe?) and the file that people should be able to refer to to see at a quick glance what is necessary for the form. The role and type are required fields for non-MCF data, so we should be using yup conditionals here to render them as required only when the condition `not MCF` is met.